lib/activerecords/form.php

```diff
@@ -330,3 +338,35 @@ public function __construct(\Icybee\Modules\Forms\Form $target, array $payload)
 		parent::__construct($target, 'render', $payload);
 	}
 }
+
+/**
+ * Event class for the `Icybee\Modules\Forms\Form::render:after` event.
+ */
+class AfterRenderEvent extends \ICanBoogie\Event
```

Comment:
Could you rename the class as `RenderCompleteEvent`? We don't "after", although we use "before".